### ISSUE TITLE
Leadership Cards should navigate as one tab for shortcut use

### ISSUE THREAD

**Issue by benhammondmusic** [SatcherInstitute/health-equity-tracker#1039]
Reading the About Us > Our Team page

Currently for Leadership entries, the image alt text, name and role are all separate tab-able entries. For keyboard users and clarity, these should be read aloud as a single entry to allow quicker tabbing

Suggestion to product team from a11y issues spreadsheet: " issue w.r.t focus grouping. set individual group focus for respective 'leadership team individual image + Name + title or designation' "
